packages/maven/sentry-android-gradle-plugin/1.7.36.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "io.sentry:sentry-android-gradle-plugin",
+    "canonical": "maven:io.sentry:sentry-android-gradle-plugin",
+    "version": "1.7.36",
+    "package_url": "https://search.maven.org/artifact/io.sentry/sentry-android-gradle-plugin",
+    "repo_url": "https://github.com/getsentry/sentry-android-gradle-plugin",
+    "main_docs_url": "https://docs.sentry.io/platforms/android"
```

Comment:
A more specific link would be https://docs.sentry.io/platforms/android/proguard/

But again not sure this is stable/correct given we might move this around, or do more with the plugin. So the top level android sounds better.

Reply:
it'd make sense to have its own "Plugin page" that could be under multiple platforms cus the plugin is used by Flutter/RN/Cordova etc

Reply:
lets keep the android root page, agreed